index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <style>
+   body {
+     background: rgb(255, 236, 29);
+   }
+   .picture {
+     size: 300px;
+     text-align: center;
+     margin: 40px auto;
+   }
+   h1 {
+     font-size: 42px;
+     color:rgb(255, 29, 82)
+   }
+
+   ul {
+     font-size: 30px;
+     text-align: left;
+   }
+   li {
+     color: black;
+   }
+  </style>
+</head>
+
+<body>
+    <img class="picture" src="https://cdn.mos.cms.futurecdn.net/42E9as7NaTaAi4A6JcuFwG-1024-80.jpg" alt="bananas">
```

Comment:
See https://www.w3schools.com/css/css_rwd_images.asp to make this image scale with the browser. Otherwise the image gets clipped as the browser window shrinks.